### ISSUE TITLE
#tempName

### ISSUE THREAD

**Issue by dlschwartz** [srophe/e-gedsh#117]
@upossekel and I are looking at the xml and neither of us knows where the ref attribute "#tempName-[number]" comes from. There are 41 instances in the xml (search "#tempName"). Did we run a script that put these in? Thanks.

**Comment by wsalesky** [comment]
I do not remember adding these.

**Comment by dlschwartz** [comment]
@davidamichelson Any idea where these came from?

**Comment by dlschwartz** [comment]
@davidamichelson Any idea where these came from? I think we want to delete but please let me know if that's not the case.

**Comment by dlschwartz** [comment]
There were a number of oddities with these tempNames. Some were attached to orgName elements. Syriaca probably should have a place to mint URIs for organizations but we don't. here I kept the orgName element but removed any attributes. For placeNames, wherever possible I have replaced them with Syriaca.org URIs. Given that there were inconsistencies with how these tempNames were assigned to placeName elements, I have removed the related attributes and replaced this with a note indicating that we need a URI.
